### ISSUE TITLE
Headless screenshot times out on an active url

### ISSUE THREAD

**Issue by Tahir52** [projectdiscovery/nuclei#4166]
### Current Behavior:
Target URL: https://185.239.164.101
Running the command: `./nuclei -id screenshot -page-timeout 30 -u https://185.239.164.101:443 -headless -debug` returns the following output:

> [WRN] [screenshot] Could not execute request for https://185.239.164.101:443: could get html element: error occurred
> executing action: [:RUNTIME] could not navigate to url https://185.239.164.101:443 <- context deadline exceeded

The screenshot is not captured, however the URL is active and serving content.

### Expected Behavior:
The screenshot should be captured in the respective file path.

### Steps To Reproduce:
- Run: `./nuclei -id screenshot -page-timeout 30 -u https://185.239.164.101:443 -headless -debug`
> NOTE: increasing the timeout doesn't change outcome.

**Comment by MetzinAround** [comment]
I'm pretty surprised to see a context deadline exceeded when you already set the page-timeout to 30. 

Could you try it with the flag  `-timeout` set to something higher than 10 as well? That might help.

**Comment by Tahir52** [comment]
After running:
`./nuclei -id screenshot -page-timeout 30 -u https://185.239.164.101:443 -timeout 30 -headless -debug`

I still get:
`[WRN] [screenshot] Could not execute request for https://185.239.164.101:443: could get html element: error occurred executing action: [:RUNTIME] could not navigate to url https://185.239.164.101:443 <- context deadline exceeded`

**Comment by Mzack9999** [comment]
This particular host only wants `en` as `Accept-Language` header:
``
...
Accept-Language: en;
...
``

**Comment by Tahir52** [comment]
@Mzack9999 thank you! That fixes the problem.
Can you kindly share the debugging process? I have more cases like this to verify.

**Comment by Mzack9999** [comment]
@Tahir52 Unfortunately there isn't a generic way of debugging. The process I followed for this case is the following:
- I verified that a normal browser could navigate correctly to the target. Then I captured the request with a proxy.
- I captured the same request sent from nuclei starting it with the `-proxy` option
- By comparing the two requests I noticed that the header was different, and by removing the two additional values, things started to work

Hope this was helpful!
